composer.json

```diff
@@ -10,7 +10,9 @@
         { "name": "Jérôme Tamarelle", "email": "jerome.tamarelle@mongodb.com" }
     ],
     "require": {
-        "php": "^8.1"
+        "php": "^8.1",
+        "ext-mongodb": "^1.17.0",
```

Comment:
Ah, it just occurred to me that extension version 1.17-dev is required because we're depending on the dev branch of PHPLIB. This likely answers my earlier questions about the CI configs, but I'll defer to you to resolve those (no comment needed).